go/libkb/home.go

```diff
@@ -65,12 +79,28 @@ func (x XdgPosix) MobileSharedHome(emptyOk bool) string {
 
 func (x XdgPosix) dirHelper(env string, prefixDirs ...string) string {
 	var prfx string
-	prfx = os.Getenv(env)
-	if len(prfx) == 0 {
+
+	// If the use explicitly provided a `--home` directory, it overrides any XDG_*
+	// variables. All XDR dirs are taken relative to that.
+	if x.getHomeFromCmd != nil {
+		prfx = x.getHomeFromCmd()
```

Comment:
So if we pass in a home directory ~/a/b/c, won't `prefixDirs` not be appended to it since the check is if `prfx == ""` on line 98?

Reply:
Good point, that's a bug.  Will fix